### DISCUSSION TITLE
Asked by Hildebrandt to renew consent, needed to reset password, now can't find where to put it in HA

### DISCUSSION THREAD

**Discussion by MHKnowles** [HandyHat/ha-hildebrandglow-dcc#402] [General]
Hi there,
I was today asked by Hildebrandt to renew consent to view my meters.  That required me to log in to their Bright app.  I didn't have my password noted down which I used previously so I had to reset it.  That all went swimmingly but this HA integration now fails with authentication failure.  I expected this because i've clearly changed the password it uses, but nowhere in the UI can I find out where to input a new password.  Any ideas?

**Comment by TotalSpaceshipguy** [comment]
Hi

I had exactly this problem today. After much digging through far too much source code, I found that Home Assistant stores the config details in a file  called "core.config_entries" and this lives in its ".homeassistant/.storage" folder (note the leading full stops as these folders will be hidden)

I run a core install of Home Assistant so this folder structure is under the "/home/homeassistant" folder for me. If you use the OS or container installs this may  be elsewhere. Perhaps you could search the file system for that "core.config_entries" file.

I guess you may also be able to remove and re-add the integration which might allow you to reconfigure. I wasn't sure if doing that would also remove the related energy readings from the database as well, and I didn't want that to happen.

**Comment by MHKnowles** [comment]
You sir are a gentleman and a scholar (and dare I say it, an acrobat and a rinky-dink panther to boot!)

I have managed to find and edit that file and my Hildebrand integration is now operational again.

For completeness, my file was in “homeassistant/.storage” (i.e. the root homeassistant folder wasn’t hidden).  This is HAOS.

Many thanks!
Mike.
